language_bindings/rust/README.md

```diff
@@ -0,0 +1,62 @@
+# halide rust bindings
+
+
+These instructions also assume a fresh install of Ubntu LTS 21.04.
+
+First, update your system.
```

Comment:
These instructions should duplicate the top-level README, but rather link to the appropriate section.